### ISSUE TITLE
Could I change the loss function from training to training?

### ISSUE THREAD

**Issue by pedro-alonsod** [ThilinaRajapakse/simpletransformers#1198]
If I have a regression task trained in some data is there an easy way to retrain that model for classification task now? I'm thinking that just a change in the loss function should do the trick. Going from regression to binary cross entropy, but I have found no easy way to do it. I don't need the output of the regression model that is just for pre-training for my real task.

What I have been doing is to use the regression trained model and change the classification to output labels to 0.0 and 1.0, then find a cut-off point where the test data is maximised but, this is not ideal.

So, is this possible? Thanks.

**Comment by Zhylkaaa** [comment]
you can actually try to just pass your pre-trained model to classification model, just try to load your model with 
```
from transformers import AutoModelForSequenceClassification

model = AutoModelForSequenceClassification.from_pretrained('<path_to_model>, num_labels=1)
```
If you don't see the warnings that last layer wasn't used for model initialisation that you are good and you can fine tune your model on classification task with simpletransformers.

Second scenario is just layer name differences, that you may need to hack it a bit and change the name of parameter in state dict of the model and save it with name that corresponds to classification model, and that youse scenario 1.

Also I can't actually infer what are you trying to do. Will you fine-tune your model or just want to adapt outputs? Because in second case I think you don't even need simpletransformers, just pass outputs to sigmoid function and find a threshold for positive class. 

If this isn't what you want, can you share some details?
